src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -11038,7 +11038,7 @@ long M(int i)
             Assert.DoesNotContain($"error CS0078", output, StringComparison.Ordinal);
             Assert.DoesNotContain($"warning CS0078", output, StringComparison.Ordinal);
 
-            // Diagnostic '{0}: {1}' was programmatically suppressed by a DiagnosticSuppressor with suppresion ID '{2}' and justification '{3}'
+            // Diagnostic '{0}: {1}' was programmatically suppressed by a DiagnosticSuppressor with suppression ID '{2}' and justification '{3}'
```

Comment:
Please update `SuppressionDiagnosticDescriptorMessage` in CodeAnalysisResources.resx as well.

Reply:
@cston Looks like `SuppressionDiagnosticDescriptorMessage` in CodeAnalysisResources.resx is already correct.

Reply:
Looks like whenever the resx file was updated, the designer for it wasn't updated to match. AFAICT it's all consistent now with this PR. 👍